### PR TITLE
Fix bug and Add music album feature

### PR DESCRIPTION
**Pull request by fredo509**
Hello 

For this milestone I 

- [x] -Refractor music and genre class
- [x] -Update design of music render
- [x] - Allow user to add authors and genre from an existing list 
- [x] --Add genre as one to many relationship 
- [x] Check feature

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ This project involves creating a console application that allows users to keep t
 
 ## 🚀 Live Demo <a name="live-demo"></a>
```
Comment (mwanawabangona) [inline README.md:82]: - [x] `Optional`
- [x] 👋 If the live demo is not available please label it not available than it going to an empty link it looks none professional 🥂

File: README.md [hunk 2]
```diff
@@ -81,7 +81,7 @@ This project involves creating a console application that allows users to keep t
...
 
-- [Video Presentation](https://drive.google.com/file/d/13-B45_cwSHZzMrKYeY-2BehYVFYm5_Jb/view?usp=sharing)
+- [Video Presentation](https://drive.google.com/file/d/1SF1Gu6ZIyrdLGXMXFn6UKtLFpfNdF_yK/view?usp=drive_link)
```
Comment (mwanawabangona) [inline README.md]: - [x] `required`
- [x] Amazing working so far ! 🎆 
- [x] However, the presentation requires all members in the video to talk about the project. 
- [x] I suggest all members should talk about the project and don't forget to talk about what made you proud the most. 😜

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi @Team!,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits
* ✅  Tests working fine


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — APPROVED** [review]
Hi @Team, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**